### PR TITLE
feat : added feature to add bookmark

### PR DESCRIPTION
**Pull request by anupkgurung**
Added bookmark feature 
user can add bookmark 
user can delete bookmark

### CODE REVIEW

**Review by vermasavita — APPROVED** [review]
Looks good to merge.

**Review by maitrakhatri — APPROVED** [review]
Clean Code. Easy to read

File: src/features/User/userSlice.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            state.isLoading=true
+        })
+        .addCase(getAllUser.fulfilled,(state,{payload})=>{
+            state.isLoading = false
+            state.user = payload?.data.users
+        })
+        .addCase(getAllUser.rejected,(state,{payload})=>{
+            state.error = payload?.response?.data?.errors[0]
+        })
+        //get all bookmarks
```
Comment (vaibhav18matere) [inline src/features/User/userSlice.js:59]: remove comments

File: src/component/UserFeed/UserFeed.jsx [hunk 2]
```diff
@@ -96,7 +106,7 @@ export const UserFeed = ({ post }) => {
                         <div className="flex items-center w-16">
                             <button className="w-10 h-10 flex items-center justify-center rounded-full hover:text-blue-500 hover:bg-blue-100"
                                 onClick={()=>handleLikePost(post._id)}>
-                                <span className={`${hasLike ? "material-icons-outlined":"material-icons-outlined"} text-xl sm:text-[22px]`}>favorite_border</span>
+                                <span className="material-icons-outlined text-xl sm:text-[22px]">{hasLike ? 'favorite' : 'favorite_border'}</span>
```
Comment (vaibhav18matere) [inline src/component/UserFeed/UserFeed.jsx:101]: spell check `favourite`

**Review by vaibhav18matere — APPROVED** [review]
looks good to merge:
tiny improvements below

**Review by MarkVed17 — APPROVED** [review]
Would've been even better to see a deploy preview. Reading just the code, I guess it should work.

**Pull request merged**